src/client/aliasviewer.py

```diff
@@ -120,22 +120,22 @@ def nick_times(self, times):
         current_times = [t for t in times if t['time'] is None]
 
         past_times.sort(key=lambda t: t['time'])
-        name_format = "<b>{}</b>"
-        past_format = "until {}"
+        name_format = "{}"
+        past_format = "{}"
```

Comment:
While we're at it, these could be removed, since they don't format anything anymore.

Reply:
I was worried you never ask .. just a sec... ;-)

Reply:
done